src/main/algorithms/search/algorithms/breadthSearch.ts

```diff
@@ -0,0 +1,111 @@
+import type { SearchAlgorithmImplementation, SearchSimulation, SearchSimulationStep } from '@/main/algorithms/search/algorithms/types'
+import type { Graph } from '@/main/algorithms/search/graph/graph'
+import type { Coords, EdgeValue, GraphFormGrid, VertexValue } from '@/main/algorithms/search/graphForm/types'
+import type { Vertex } from '@/main/algorithms/search/graph/vertex'
+import { ProtocolBuilder } from '@/main/simulation/protocolBuilder'
+import { GraphFormItem } from '@/main/algorithms/search/graphForm/graphFormItem'
+import type { Edge } from '@/main/algorithms/search/graph/edge'
+
```

Comment:
BFS does not show end vertex in simulation

Reply:
fixed